src/element/bounds.ts

```diff
@@ -220,12 +220,11 @@ export const getArrowheadPoints = (
 
   const length = element.points.reduce((total, [cx, cy], idx, points) => {
     const [px, py] = idx > 0 ? points[idx - 1] : [0, 0];
-    return total + Math.hypot(cx - px, cy - py);
+    return Math.hypot(cx - px, cy - py);
   }, 0);
```

Comment:
can we remove the reduce and just use the last two points for better perf?

Reply:
Yes! The changes have been pushed